pyramid_oereb/contrib/sources/document.py

```diff
@@ -183,20 +193,21 @@ def _get_mapped_value(self, document, key, multilingual=False):
             if attribute:
                 value = getattr(document, attribute)
                 if value:
-                    return {self._language: value} if multilingual else value
+                    return {language: value} if language else value
         return None
 
-    def _get_document_title(self, document, current_file):
+    def _get_document_title(self, document, current_file, language):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/app/pyramid_oereb/pyramid_oereb/pullRequest?prid=4491046)